tests/unit/Rules/UniqueTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+/**
+ * @group  rule
+ * @covers \Respect\Validation\Rules\Unique
+ */
+class UniqueTest extends RuleTestCase
+{
+    public function providerForValidInput()
+    {
+        $rule = new Unique();
+
+        return [
+            [$rule, []],
+            [$rule, [1, 2, 3]],
```

Comment:
Would be nice to create tests with different kinds of data (arrays, booleans, objects, floats, strings)